components/VotingPower/DelegateMenu.jsx

```diff
@@ -0,0 +1,120 @@
+import {
+  Button, Divider, Typography, Form, Input, notification,
+} from 'antd';
+import PropTypes from 'prop-types';
+import { useState } from 'react';
+import { isValidAddress } from '@autonolas/frontend-library';
+import { useHelpers } from 'common-util/hooks/useHelpers';
+import { truncateAddress } from 'common-util/functions';
+import { StyledMenu } from './styles';
+import { formatWeiBalanceWithCommas } from './utils';
+import { useDelegatorList } from './hooks';
+import { delegate } from './requests';
+
+const { Text, Paragraph } = Typography;
+
+const DelegateMenu = (props) => {
+  const [form] = Form.useForm();
+  const { account } = useHelpers();
+  const { delegatorList } = useDelegatorList(account);
+
+  const [delegateddAddress, setDelegatedAddress] = useState(null);
+  const [delegateFormVisible, setDelegateFormVisible] = useState(false);
+  const [isSending, setIsSending] = useState(false);
+
+  const handleSubmit = async (values) => {
+    setIsSending(true);
+    setDelegatedAddress(null);
+
+    try {
+      const { address } = values;
+      await delegate({ delegatee: address });
+      setDelegatedAddress(address);
+      form.resetFields();
+      notification.success({
+        message: 'Delegated voting power',
+      });
+      setDelegateFormVisible(false);
+    } catch (error) {
+      console.error(error);
+      notification.error({
+        message: 'Couldn’t delegate',
+      });
+    } finally {
+      setIsSending(false);
+    }
+  };
+
+  return (
+    <StyledMenu className="p-12">
+      <Text strong>Your voting power</Text>
+      <Paragraph className="my-8">
+        {formatWeiBalanceWithCommas(props.balance)}
+      </Paragraph>
+      <Divider className="my-8" />
+      <Text strong>You’ve delegated to:</Text>
+      <Paragraph className="my-8">
```

Comment:
we should either:
- link to etherscan page of address
- add some some tooltip to the address so the user can see the full address